_posts/0000-01-13-ftanquary.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![picard-facepalm](https://cloud.githubusercontent.com/assets/16547949/25401008/6f0a5f0c-29c2-11e7-8ade-705cc5e57333.jpg)
 {: .center}
 
-Caption here
+You're making my brain cry!
```

Comment:
@ftanquary Funny. Please add a comment so I can approve the PR.

Reply:
this is a good first change but you need more changes.  change is constant.

Reply:
Please approve - that would be fantastic